### PR TITLE
event model

### PR DESCRIPTION
**Pull request by Parikshit-007**
#57

### CODE REVIEW

**Comment by kskarthik5** [comment]
please add the attendance attribute in the schema

File: models/event.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { connector } = require("./databaseUtil");
+
+const eventSchema = new connector.Schema({
+    eventName: { type: String, required: true },
+    date: { type: Date, required: true },
+    infra: { type: String, required: true },
```
Comment (kskarthik5) [inline models/event.js:6]: infra will be of type:Infrastructure and add attendance attribute which be be of type:[Attendance]

Comment (kskarthik5) [reply models/event.js:6]: put the necessary imports as well

**Review by kskarthik5 — CHANGES_REQUESTED** [review]

**Comment by Hitansh159** [comment]
With the new Plan, this model would be redesigned so closing this PR.

**Pull request closed without merging**